release/models/wifi/openconfig-wifi-types.yang

```diff
@@ -248,6 +254,27 @@ module openconfig-wifi-types {
       each frequency.";
   }
 
+  identity FREQ_6GHZ {
+    base OPERATING_FREQUENCY;
+    description "The Radio or SSID will operate at 6GHz only.";
+  }
+
+  identity FREQ_5_6GHZ {
+    base OPERATING_FREQUENCY;
+    description "The Radio or SSID will be dual-band; operating in both 5 &
+    6GHz frequencies. Dual-band Radio typically refers to a Monitor-mode
+    radio, hopping between frequencies, dwelling for a configurable amount of
+    time on each frequency.";
+  }
+
+  identity FREQ_2_5_6GHZ {
```

Comment:
Can we add space before "GHZ" ? because for 2 and 5 we have FREQ_2_5_GHZ

Reply:
I have updated  from FREQ_2_5_6GHZ to FREQ_2_5_6_GHZ. I have also updated FREQ_5_6GHZ to FREQ_5_6_GHZ.